openeogeotrellis/deploy/batch_job.py

```diff
@@ -509,11 +509,10 @@ def write_metadata(metadata, metadata_file, job_dir: Path):
             s3_instance = s3_client()
 
             logger.info("Writing results to object storage")
-            for filename in os.listdir(job_dir):
-                if filename == UDF_PYTHON_DEPENDENCIES_FOLDER_NAME:
-                    logger.warning(f"Omitting {filename} as the executors will not be able to access it")
+            for file_path in filter(lambda x: x.is_file(), job_dir.rglob("*")):
```

Comment:
As discussed: the change from simple listing to recursive glob might have unintended impact. Note that `job_dir` might contain more than just the batch job results a user is interested in, e.g. a large file tree of UDF dependencies or other intermediate/auxiliary resources